cert/go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/lightningnetwork/lnd/cert
 
 go 1.19
 
+require github.com/stretchr/testify v1.5.1
```

Comment:
nit: commit message should probably read "build: run go mod tidy"?

Reply:
I should have escaped the \` from git commit -m "build: run \`go mod tidy\`" :+1: